tests/cypress/e2e/jcontent/selection.cy.ts

```diff
@@ -224,7 +224,7 @@ describe('Multi-selection tests', {testIsolation: false}, () => {
             jcontent.getTable().getRowByLabel('We are a global network');
         });
 
-        it('remove selection when navigating to a mode where item is not visible', () => {
+        it.skip('remove selection when navigating to a mode where item is not visible', () => {
```

Comment:
@AKarmanov I see that you already fixed selection here https://github.com/Jahia/jcontent/pull/1372/files?w=1#diff-6ce08d252f22a7856351f39ae758ab490db676f17a43276052c65a1680ad38c0R309

So maybe we just need to remove skip here and check if it's passing again